client/js/socket-events/init.js

```diff
@@ -4,12 +4,18 @@ const $ = require("jquery");
 const escape = require("css.escape");
 const socket = require("../socket");
 const render = require("../render");
+const options = require("../options");
 const webpush = require("../webpush");
 const sidebar = $("#sidebar");
 const storage = require("../localStorage");
 const utils = require("../utils");
 
 socket.on("init", function(data) {
+	if (options.initialized) {
```

Comment:
I'm quite certain doing this breaks reconnections.

Reply:
This is already done in a very similar way currently, are you sure about that?

Reply:
Your change blocks future `init` calls, which happen after reconnecting. On master it blocks `configuration` which is fine.